public/js/app/util/__test__/helpers.test.js

```diff
@@ -5,58 +5,42 @@ const coursesTypeOne = {
   course_SV: {
     courseCode: 'ML1616',
     gradeScaleCode: 'AF',
-    title: { sv: 'Industriell projektledning och projektstyrning' },
-    credits: 7.5,
+    title: 'Industriell projektledning och projektstyrning',
+    credits: { number: '7.5' },
     creditUnitLabel: 'Högskolepoäng',
     creditUnitAbbr: 'hp',
     courseLiterature: '<p>Meddelas vid kursstart.</p>',
   },
   course_EN: {
     courseCode: 'DD1321',
     gradeScaleCode: 'AF',
-    title: { en: 'Applied Programming and Computer Science' },
-    credits: 7.5,
+    title: 'Applied Programming and Computer Science',
+    credits: { number: '7.5' },
     creditUnitLabel: 'Credits',
     creditUnitAbbr: 'hp',
   },
 }
-const courseTypeTwo = {
-  course: {
-    title: {
-      sv: 'Tillämpad programmering och datalogi',
-      en: 'Applied Programming and Computer Science',
-    },
-    credits: 9,
-    creditUnitLabel: {
-      sv: 'Högskolepoäng',
-      en: 'Credits',
-    },
-    creditUnitAbbr: {
-      sv: 'hp',
-      en: 'hp',
-    },
-  },
-}
+
 describe('combine course name', () => {
   test('combine course name for an object from a detailed information endpoint in Swedish', () => {
     const courseTitle = combinedCourseName('ML1616', coursesTypeOne.course_SV, 'sv')
-    expect(courseTitle).toBe('ML1616 Industriell projektledning och projektstyrning 7,5 hp')
+    expect(courseTitle).toBe('ML1616 Industriell projektledning och projektstyrning 7.5 hp')
```

Comment:
This should be `7,5 hp` with , in swedish, but I think it will be fixed with `formattedWithUnit` prop